app/helpers/home_helper.rb

```diff
@@ -0,0 +1,2 @@
+module HomeHelper
```

Comment:
Leftover from rebase, let's remove it

Reply:
I am soooooo sorry. I removed all mentioned above before rebase, and forgot to check it after.
Done now :)

Reply:
no need to be sorry - happens to the best of us :)